src/test/java/space/obminyashka/items_exchange/util/AdvertisementDtoCreatingUtil.java

```diff
@@ -63,4 +94,8 @@ private static AdvertisementDto getBuild(long aId, String topic, String descript
                 .images(images)
                 .build();
     }
+
+    private static String createString(int quantityOfCharsInNewsString){
+        return "x".repeat(Math.max(0, quantityOfCharsInNewsString));
```

Comment:
Could you explain what it the cause of creation such method?

Reply:
i need to check validation, where field can`t contain more then a certain amounts of chars. for example, adverisement dto descriprion  must be less than  255  symbols